baseline/pytorch/lm/train.py

```diff
@@ -46,15 +47,16 @@ def _get_pytorch_model(self):
         return self.model.module if self.gpus > 1 else self.model
 
     @staticmethod
-    def _get_dims(batch_dict):
+    def _get_dims(loader):
```

Comment:
Not related to this PR because the problem is already there but some of the tf trainers call this `_get_batchsz` we should be consistent

Reply:
nvm I see now that that it can be used for dims other than batch size like when getting the number of tokens